### ISSUE TITLE
I need help with function

### ISSUE THREAD

**Issue by Raserad** [daybrush/moveable#795]
@daybrush hello. Can you explain me how you to render this blue rectangle depends on gray divs

![image](https://user-images.githubusercontent.com/34380690/202520077-dd0513be-6720-4369-9695-f2e5ef70f713.png)

I need create such div with rotation which bounds gray divs with

**Comment by daybrush** [comment]
@Raserad 
There is a defaultGroupRotate option.


set `defaultGroupRotate={40}`

**Comment by Raserad** [comment]
> @Raserad There is a defaultGroupRotate option.
> 
> set `defaultGroupRotate={40}`

Yes i know. I asked about how to render such rectangle by pure js? How to find coordinates such rectangle?

**Comment by daybrush** [comment]
@Raserad 

You can set by setter
```js
moveable.defaultGroupRoate = 40;
```



getRect has pos1, pos2, pos3, pos4 information and rotation information.

https://daybrush.com/moveable/release/latest/doc/Moveable.html#.RectInfo

```
moveable.getRect();
```

**Comment by jadiagaurang** [comment]
@daybrush, I am trying to do the same as @Raserad asked earlier. When I group multiple targets into a group. I need to show the Rectangle to visualize the group without initializing Moveable.

I need your help to use the `getRect()` function to create an HTML Element that shows borders around grouped elements.

I have explained more in detail on this issue. https://github.com/daybrush/moveable/issues/1004

The function I wrote which works when elements are not rotated. But, as soon as I rotate elements, the calculation doesn't work.

Any help is appreciated 🙏🏻

**Comment by jadiagaurang** [comment]
Hi @Raserad,

I am looking for something similar. Have you resolved the issue? If yes, please let me know how you resolved the issue.
